pep_sphinx_extensions/pep_theme/templates/redirect.html

```diff
@@ -0,0 +1,16 @@
+{# Redirect template #}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="refresh" content="0; url={{ pathto(pep_file) }}">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
```suggestion
```

Elide unnecessary element for page with no content that we want to load, parse, render and redirect as fast as possible.